packages/local_auth/pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: Flutter plugin for Android and iOS device authentication sensors
   such as Fingerprint Reader and Touch ID.
 author: Flutter Team <flutter-dev@googlegroups.com>
 homepage: https://github.com/flutter/plugins/tree/master/packages/local_auth
-version: 0.6.0+2
+version: 0.6.0+3
```

Comment:
I believe for API changes, we up the minor version. So this should be 0.6.1

Reply:
Resolved.